sql/src/main/java/org/apache/druid/sql/calcite/expression/builtin/MultiValueStringOperatorConversions.java

```diff
@@ -403,14 +402,12 @@ public DruidExpression toDruidExpression(
           plannerContext,
           rowSignature,
           rexNode,
-          druidExpressions -> {
-            final List<DruidExpression> newArgs = harmonizeNullsMvdArg0OperandList(druidExpressions);
-            return DruidExpression.ofFunctionCall(
-                Calcites.getColumnTypeForRelDataType(rexNode.getType()),
-                getDruidFunctionName(),
-                newArgs
-            );
-          }
+          druidExpressions ->
```

Comment:
this looks like the default implementation, so can just delete i think, unless i'm missing something?

Reply:
I think you're right, will delete it.

Reply:
ah, sorry for confusion, i meant of the entire `toDruidExpression ` method override can be dropped i think, since this is effectively the same as `DirectOperatorConversion` implementation

Reply:
Ah, I had deleted the other one, not this one you commented on. Just deleted this one too. Actually, I did a bigger change:

- updated MV Contains and Overlap to call a superclass constructor with their operator and function names, rather than overriding `calciteOperator()` and `getDruidFunctionName()`
- removed `toDruidExpression` and `toDruidExpressionWithPostAggOperands` from MV Contains and Overlap
- marked `calciteOperator()` and `getDruidFunctionName()` as `final` in `DirectOperatorConversion`, so all the *other* methods on `DirectOperatorConversion` can be trusted to use the correct operator and function name